<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuggestAnAppPageContentWidget</class>
 <widget class="QWidget" name="SuggestAnAppPageContentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QVBoxLayout" name="contentLayout">
       <item>
        <widget class="QLabel" name="appNameLabel">
         <property name="text">
          <string>App name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="appNameLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="urlLabel">
         <property name="text">
          <string>Website / download page:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="urlLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="summaryLabel">
         <property name="text">
          <string>A short summary, why is it so awesome:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="summaryTextEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
